SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -355,22 +355,24 @@ QueryConstraint InterestFactory::CreateAlwaysInterestedConstraint() const
 }
 
 
-QueryConstraint InterestFactory::CreateSingletonConstraint() const
+QueryConstraint InterestFactory::CreateAlwaysRelevantConstraint() const
 {
-	QueryConstraint SingletonConstraint;
+	QueryConstraint AlwaysRelevantConstraint;
 
-	Worker_ComponentId SingletonComponentIds[] = {
+	Worker_ComponentId ComponentIds[] = {
 		SpatialConstants::SINGLETON_COMPONENT_ID,
-		SpatialConstants::SINGLETON_MANAGER_COMPONENT_ID };
+		SpatialConstants::SINGLETON_MANAGER_COMPONENT_ID,
```

Comment:
maybe not worth it now, but we discussed removing the manager from this list a while back.

Reply:
Yep, we can do that when we finally remove the streaming query also.